LondonBritishLibrary/orient/BLorient501.xml

```diff
@@ -310,23 +306,68 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-        
-    </decoNote>
+                         <locus target="#1r #62r #70r #79v #97r #123r #177r"/>
+                         <desc>The commencement of each book is decorated with a frame in the usual 15th century style.</desc>
+                     </decoNote>
                       <decoNote xml:id="d2" type="ornamentation">
                           <desc>Most pages are decorated with several crux ansata with coronis.</desc>
                       </decoNote>
+                      <decoNote xml:id="d3" type="rubrication">
+                          <desc>Crux ansata, headlines, numbers and punctuation marks.</desc>
+                      </decoNote>
+                      <decoNote xml:id="d4" type="band">
+                          <locus target="#86vb #92rb #99rb #101va #104rb #109rb #119va"/>
+                          <desc>The commencement of each book is decorated with a band in the usual 15th century style.</desc>
+                      </decoNote>
+                      <decoNote xml:id="d5" type="band">
+                          <locus target="#194va"/>
+                          <desc>Humble band by a later hand.</desc>
+                      </decoNote>
                   </decoDesc>
-                        
-                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#194vb"/>
+                                    <desc type="Unclear">Erased and partially illegible, incomprehensible text by a poorly trained hand, possibly with magical 
+                                        content.</desc>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus from="194vb" to="195ra"/>
+                                    <desc type="Unclear">Incomprehensible text by a poorly trained hand, possibly with magical content.</desc>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#1r #195v"/>
+                                    <desc type="StampExlibris">Small round stamp with coat of arms, lion, unicorn and inscription: 'British Museum'.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#5v #9v #15v #21v #27v"/>
+                                    <desc type="StampExlibris">Small round stamp with crown and inscription: 'British Museum' on every fourth to every sixth folio.</desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#2r"/>
+                                    <desc type="findingAid">Note by a later Ethiopian hand indicating the main text: <foreign xml:lang="gez">ኢሳይያስ፡
```

Comment:
If it is written in the same style as the same note in other manuscripts of Magdala, it might be marked up with the church's ID (examples can be found in the other British Library manuscripts). findingAis makes sense, in the other manuscripts this note has been categorised as OwnershipNote

Reply:
There are 12 repositories carrying the name መድኃኔ፡ ዓለም at the moment, but I this one should be INS0101MadhaneAlam, this is Madḫāne ʿAlam of Maqdala. Who is ኢሳይያስ then? Probably it is a person with the name ኢሳይያስ, who was a monch from a monastary with the name መድኃኔ፡ ዓለም. In this case, I need to create a PRS ID for ኢሳይያስ? 
INS0101MadhaneAlam is also encoded for: 
BLorient781 
BLorient623
BLorient748
BLorient798

Reply:
Can I encode it as:
`<placeName ref="INS0101MadhaneAlam">መድኃኒ፡ ዓለም</placeName>`

Reply:
Perhaps it is PRS13299Isayeyyas, who also owned BLorient788

Reply:
PRS13080Isayeyyas has owned BLorient539. Probably they are all the same person?

Reply:
PRS11364Isayeyas has owned BLorient489
PRS11385Isayeyya has owned BLorient494
PRS12261Isayeyyas has owned BLorient642
PRS13704Isayeyyas has owned BLorient674
I think, the probability is high, that it is always the same Isayəyyās, even though the subscriptions are written with different hands. This name is not very frequent among scribes and owners.

Reply:
Since Isayəyyās is a frequent name it is difficult to say for sure that it is always the same one, unless we have other indications (names of relatives, precise time span and place)

Reply:
`Since Isayəyyās is a frequent name it is difficult to say for sure that it is always the same one, unless we have other indications (names of relatives, precise time span and place)`
In my opinion Isāyəyyās is not a frequent name. I found twelve records with this personal name. Two are persons from outside Ethiopia (the prophet and an abbot from Egypt). From the remaining 10, there are seven related to Magdala manuscripts. They are all noted as owners or donors who ordered the manuscripts. I think, it is very likely, that all of them or at least some of them are the same person, unless this is not certain. At least time and place are documented to be similar. I think, it is enough evidence to make a relation to indicate, that they are possibly the same persons. It is probably not enough evidence to merge the IDs.